scripts/generateDist.js

```diff
@@ -0,0 +1,38 @@
+// const fs = require('fs');
+const path = require('path');
+// const webpack = require('webpack');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+
+// const jsonContents = fs.readFileSync(path.join(__dirname, '..', 'package.json'));
+// const json = JSON.parse(jsonContents);
+// const version = json['version'];
+
+module.exports = {
+  entry: {
+    'log-weasel': path.join(__dirname, 'distTemplate.js'),
+    'log-weasel.min': path.join(__dirname, 'distTemplate.js')
+  },
+  output: {
+    path: path.join(__dirname, '..', 'dist'),
+    filename: '[name].js',
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        use: 'babel-loader',
+      },
+    ]
+  },
+
+  devtool: 'source-map',
+
+  plugins: [
+    new UglifyJSPlugin({
+      sourceMap: true,
+      include: /\.min\.js$/
+    })
```

Comment:
https://www.npmjs.com/package/terser is the modern, ESNext aware version of uglify. Want to use it here?